src/main/kotlin/dev/stalla/dom/DomParsingExtensions.kt

```diff
@@ -79,25 +81,17 @@ internal fun Node.parseAsTemporalAccessor(): TemporalAccessor? = DateParser.pars
  * @return The [imageBuilder] populated with the DOM node contents.
  */
 @InternalApi
+@Suppress("ComplexMethod")
```

Comment:
Also suppressed ComplexMethod in builders. There is no really good alternative to shorten down the `when` cases for namespaces with a lot of elements.

Reply:
Unpopular opinion, but ComplexMethod and its siblings are counterproductive in some cases, this being a good example of one.